cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -56,8 +58,34 @@ define([
             // It sends `POST` request, because the model doesn't have `id`. In
             // this case, it is considered to be new according to Backbone documentation.
             AjaxHelpers.expectJsonRequest(
-                requests, 'POST', '/course-details', expectedJson
+                requests, 'POST', urlRoot, expectedJson
             );
         });
+
+        it('Creating an entrance exam with minimum_score_pct. It should save the ' +
```

Comment:
This name is too long! I'd go with something shorter that expresses the intent without all the details. Something like:

    it('should save entrance exam information correctly', function () {